pages/linux/csplit.md

```diff
@@ -0,0 +1,19 @@
+# csplit
+
+> Split a file into pieces.
```

Comment:
We should add a sub-heading - "This generates files named "xx00", "xx01" and so on."

Reply:
Added in 4aa995b31953